src/css/main.css

```diff
@@ -1,3 +1,31 @@
 .no-underline {
   text-decoration: none !important;
 }
+
+.icon {
+  max-width: 25px;
+}
+
+.icon-build {
+  max-width: 20px;
+}
+
+.loader {
+  display: inline-block;
+  font-size: initial;
+  box-sizing: border-box;
+}
+
+@keyframes spin {
```

Comment:
I thought we had autoprefixer running on build time hence why I removed the vendor prefixes. Apparently we don't. We should run it so that we have one less thing to worry about.

Reply:
Along with the upstream `.browserslistrc` which we can read and thus reuse it.